### ISSUE TITLE
Register library with pip for easy installation

### ISSUE THREAD

**Issue by gabegaster** [mdezube/sms-analysis#3]
breaks on version  '0.12.0'

**Comment by mdezube** [comment]
Good point, I'm not sure what the minimum I support is offhand.  Did you test it on 12 and it broke?  If so do you have a sense what version it runs on at a minimum?  We can then add this to the pip setup like you suggested to make it all work.

**Comment by gabegaster** [comment]
Yes it broke on 0.12. I upgraded pip in a virtual environment and it worked fine.

gabe

**Comment by mdezube** [comment]
Okay, following a suggestion from Dan, I should list all dependencies in a requirements file so you can run pip once and get it all. Want to do that?  If not, I'll try to get it done this week.  Also, how would we publish to pip?

**Comment by mdezube** [comment]
Looks like you publish to pip via: http://marthall.github.io/blog/how-to-package-a-python-app/ I'll start with the requirements.txt file though and can do that later

**Comment by mdezube** [comment]
As of https://github.com/mdezube/sms-analysis/commit/c9617567887ea01db0a594e0b3c0db0c5c90f45b I've documented the requirements, I'll update this issue to: register the entire project with pip
